rapids-cmake/cpm/libcudacxx.cmake

```diff
@@ -54,6 +54,32 @@ function(rapids_cpm_libcudacxx)
   include("${rapids-cmake-dir}/cpm/detail/package_details.cmake")
   rapids_cpm_package_details(libcudacxx version repository tag shallow exclude)
 
+  set(to_install OFF)
+  if(INSTALL_EXPORT_SET IN_LIST ARGN AND NOT exclude)
```

Comment:
Parsing before hand comes with some complexity as you need to re-construct the `BUILD_EXPORT_SET` and `INSTALL_EXPORT_SET` command so they propagate to `rapids_cpm_find` so I try to avoid it as much as possbile.